python_modules/libraries/dagster-looker/dagster_looker/asset_decorator.py

```diff
@@ -1,16 +1,44 @@
 import itertools
 from pathlib import Path
-from typing import Any, Callable
+from typing import Any, Callable, Mapping, Sequence
 
+import lkml
 import yaml
 from dagster import AssetKey, AssetsDefinition, AssetSpec, multi_asset
 
 
+def build_looker_explore_specs(project_dir: Path) -> Sequence[AssetSpec]:
```

Comment:
do we plan on eventually having some sort of translator API to customize the asset key specs built?

Reply:
Yeah. That will be a separate PR